test/multiverse/suites/memcache/Envfile

```diff
@@ -1,4 +1,4 @@
-%w(2.7.6 2.7.8 2.7.11).each do |version|
+%w(2.7.6 2.7.8 2.7.11 3.0.2).each do |version|
   gemfile <<-RB
     gem 'dalli', '~> #{version}'
   RB
```

Comment:
```suggestion
%w(2.7.6 2.7.8 2.7.11).each do |version|
  gemfile <<-RB
    gem 'dalli', '~> #{version}'
  RB
end

if RUBY_VERSION >= '2.5.0'
  gemfile <<-RB
    gem 'dalli', '~> 3.0.2'
  RB
```
I didn't realize this previously, but it looks like dalli 3.0.2 only supports ruby 2.5+, which is what the current CI failures are related to. This change should make to so 3.0.2 is only running on 2.5+ and get everything passing!